src/d/actor/d_a_player_swim.inc

```diff
@@ -314,6 +521,15 @@ void daPy_lk_c::setSwimTimerStartStop() {
 }
 
 /* 8013D814-8013DA18       .text procSwimUp_init__9daPy_lk_cFi */
+/*
+ * Starts or stops the air meter timer based on Link's vertical position.
+ *
+ * If Link is significantly below the surface, the timer begins. Also sets the
+ * drowning voice clip based on air meter level and current swim animation loop.
+ * 
+ * Adjusts `m3608` field using a cosine function to reflect Link's oxygen level.
```

Comment:
This comment doesn't seem right, was this just copy-pasted from the above function?